src/features/smSystem/suppliersOrders/types/modals.ts

```diff
@@ -0,0 +1,11 @@
+import {IdName} from './index';
```

Comment:
Nie znam konkretnego miejsca użycia, nie sprawdzałem ale interfejsy takie jak te oraz te w pliku poniżej tables, raczej powinny być zaraz przy komponencie. Chyba że jest to bardzo rozbudowane wtedy ma sens to rozdzielać. 

oraz widzę żę "AddOrderModalProps" jest nie używany

PS jeśli masz do zdefiniowana komponent, i to jest jedyny komponent w pliku, interfejs może być bezpośrednio nad nim, dodatkowo wtedy z racji że jest używany tylko w tym pliku, można go nazwać "Props" bez uszczegóławiania typu "<NazwaKomponentuProps>" bo to nie ma sensu. Chyba że ten interfejs exportujemy i używamy gdzieś indziej